admin/src/scenes/plan-transport/ligne-bus/List.jsx

```diff
@@ -66,6 +71,7 @@ export default function List() {
           <Select
             options={cohortList}
             value={cohort}
+            disabled={user.role === ROLES.HEAD_CENTER ? true : false}
```

Comment:
```suggestion
            disabled={user.role === ROLES.HEAD_CENTER}
```

Un triple égal c'est déjà un boolean